Java/math/Euler's_Totient_function.java

```diff
@@ -0,0 +1,68 @@
+/*
+Euler's totient function, also known as phi-function ϕ(n), counts the
+numberof integers between 1 and n inclusive, which are coprime to n.
+*/
+import java.util.Scanner;
+
+public class Euler_Totient {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter an integer: ");
+		int n = sc.nextInt();
+
+		int temp = n;
+		//This gives euler totient function for N
+		// Time Complexity: O(N*root(N))
+		int ans = n;
+		for (int i = 2; i <= n; ++i) {
+			if (n % i == 0) {
+				ans = ans - ans / i;
+				while (n % i == 0)
+					n /= i;
+			}
+		}
+		if (n > 1) {
+			ans = ans - ans / n;
+		}
+		n = temp;
+		System.out.println("\nThe euler totient function for " + n + " is " + ans);
+
+		//This gives euler totient function from 1 to N
+		// Time Complexity: O(N loglogN) - same as Sieve of Eratosthenes
```

Comment:
can we please remove this line as we have already included it in the commented form below the code section.